debug_toolbar/toolbar.py

```diff
@@ -210,7 +219,7 @@ def __init__(self, request, get_response, request_id=None):
         self.init_store()
 
     @classmethod
-    def from_store(cls, request_id, panel_id=None):
+    def from_store(cls, request_id, panel_id=None) -> "StoredDebugToolbar":
```

Comment:
You can do the following if you include `from __future__ import annotations` at the top. As of Python 3.14, it'll be deprecated.

```suggestion
    def from_store(cls, request_id, panel_id=None) -> StoredDebugToolbar:
```